internal/node/node.bzl

```diff
@@ -139,6 +139,9 @@ def _to_execroot_path(ctx, file):
 
     return ("<ERROR> _to_execroot_path not yet implemented for " + file.path)
 
+def _strip_external(path):
+    return path[len("external/"):] if path.startswith("external/") or path.startswith("external\\") else path
```

Comment:
nit: I think we have similar logic in other places, just curious if this matches?

Reply:
Sounds right regarding slash. There was a common case in npm_package.bzl  so I factored it out `strip_external` to a new `internal/common/path_utils.bzl`.